features/FooterUpdate.feature

```diff
@@ -0,0 +1,66 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+    Background: A user arrives on the SaaSquatch help center page
+        Given the user stays on the 'https://docs.saasquatch.com/' pages
+        Then the footer categories are visible
+
+
+    @motivating
+    Scenario: Categories and links for the footer
+        Given the user looks at the footer
```

Comment:
Our standard wording around a user looking at something is `Given a user viewing the footer` , try to apply this sort of wording to the other scenarios as well